chringo/lib/main.dart

```diff
@@ -1,89 +1,137 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
 
 void main() => runApp(MyApp());
 
 final dummySnapshot = [
- {"name": "Filip", "votes": 15},
- {"name": "Abraham", "votes": 14},
- {"name": "Richard", "votes": 11},
- {"name": "Ike", "votes": 10},
- {"name": "Justin", "votes": 1},
+  {"name": "Filip", "votes": 15},
+  {"name": "Abraham", "votes": 14},
+  {"name": "Richard", "votes": 11},
+  {"name": "Ike", "votes": 10},
+  {"name": "Justin", "votes": 1},
 ];
 
 class MyApp extends StatelessWidget {
- @override
- Widget build(BuildContext context) {
-   return MaterialApp(
-     title: 'Baby Names',
-     home: MyHomePage(),
-   );
- }
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Baby Names',
+      home: MyHomePage(),
+    );
+  }
 }
 
 class MyHomePage extends StatefulWidget {
- @override
- _MyHomePageState createState() {
-   return _MyHomePageState();
- }
+  @override
+  _MyHomePageState createState() {
+    return _MyHomePageState();
+  }
 }
 
 class _MyHomePageState extends State<MyHomePage> {
- @override
- Widget build(BuildContext context) {
-   return Scaffold(
-     appBar: AppBar(title: Text('Baby Name Votes')),
-     body: _buildBody(context),
-   );
- }
-
- Widget _buildBody(BuildContext context) {
-   // TODO: get actual snapshot from Cloud Firestore
-   return _buildList(context, dummySnapshot);
- }
-
- Widget _buildList(BuildContext context, List<Map> snapshot) {
-   return ListView(
-     padding: const EdgeInsets.only(top: 20.0),
-     children: snapshot.map((data) => _buildListItem(context, data)).toList(),
-   );
- }
-
- Widget _buildListItem(BuildContext context, Map data) {
-   final record = Record.fromMap(data);
-
-   return Padding(
-     key: ValueKey(record.name),
-     padding: const EdgeInsets.symmetric(horizontal: 16.0, vertical: 8.0),
-     child: Container(
-       decoration: BoxDecoration(
-         border: Border.all(color: Colors.grey),
-         borderRadius: BorderRadius.circular(5.0),
-       ),
-       child: ListTile(
-         title: Text(record.name),
-         trailing: Text(record.votes.toString()),
-         onTap: () => print(record),
-       ),
-     ),
-   );
- }
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: Text('CHRINGO')),
+      body: _buildBody(context),
+    );
+  }
+
+  Widget _buildBody(BuildContext context) {
+    // TODO: get actual snapshot from Cloud Firestore
+    return _buildGrid(context);
+  }
+
+  Widget _buildGrid(BuildContext context) {
+    return GridView.count(
+      // Create a grid with 2 columns. If you change the scrollDirection to
+      // horizontal, this produces 2 rows.
+      crossAxisCount: 5,
+      // Generate 100 widgets that display their index in the List.
```

Comment:
Update these comments to reflect our code!

Reply:
fixed